paper/config/plugins/SimpleAdminHacks/config.yml

```diff
@@ -115,24 +115,28 @@ hacks:
       - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Check out the player made map at map.civmc.tk","color":"green","clickEvent":{"action":"open_url","value":"https://map.civmc.tk/"}}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"To join global chat, type `/nljg ! !`, to leave, type `/nlleg !`","color":"green"}]'
+      #- '["",{"text":"[CivMC] ","color":"gold"},{"text":"To join global chat, type `/nljg ! !`, to leave, type `/nlleg !`","color":"green"}]'
       - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Type /g ! to talk in global chat","color":"green"}]'
       - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Type /vote to vote for CivMC and receive essence!","color":"green"}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"The message is a lie, there are no essence streaks","color":"green"}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Lodestones work as elevators, shift to go down and spacebar to go up. They can be made in the Iron Forge factory","color":"green"}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Do not put buttons or signs over an elevator - or it will be blocked and you will not be able to teleport back to that block!","color":"green"}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Consider donating to the server at patreon.com/civ_mc!","color":"green","clickEvent":{"action":"open_url","value":"https://www.patreon.com/civ_mc"}}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"All ores except coal and copper can spawn in big veins","color":"green"}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"All ores except diamond spawn randomly as you dig","color":"green"}]'
+      #- '["",{"text":"[CivMC] ","color":"gold"},{"text":"The message is a lie, there are no essence streaks","color":"green"}]'
+      #- '["",{"text":"[CivMC] ","color":"gold"},{"text":"Lodestones work as elevators, shift to go down and spacebar to go up. They can be made in the Iron Forge factory","color":"green"}]'
+      #- '["",{"text":"[CivMC] ","color":"gold"},{"text":"Do not put buttons or signs over an elevator - or it will be blocked and you will not be able to teleport back to that block!","color":"green"}]'
+      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Consider donating to the server at patreon.com/civ_mc! To get access to some unique features!","color":"blue","clickEvent":{"action":"open_url","value":"https://www.patreon.com/civ_mc"}}]'
+      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"All ores except coal and copper can spawn in big veins","color":"green","clickEvent":{"action":"open_url","value":"https://civwiki.org/wiki/Civ101#HiddenOre"}}]'
+      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"All ores except diamond spawn randomly as you dig","color":"green","clickEvent":{"action":"open_url","value":"https://civwiki.org/wiki/Civ101#HiddenOre"}}]'
```

Comment:
There is no indicator that these messages can be clicked, maybe add a "Click for more info".

Reply:
I mostly added those links, just in case someone tried to. but sure.